examples/pl.yaml

```diff
@@ -1,3 +1,11 @@
+# Use external scheduler.  This example uses dask.distributed.Client,
+#   but any other class can be used as long as it returns a client and
+#   registers itself to dask.  The `settings` are passed as keyword
+#   arguments to the class when instantiated.
+# dask_distributed:
+#   class: !!python/name:dask.distributed.Client
```

Comment:
Are there multiple Client classes? I know there are multiple Cluster classes for the different types of clusters (slurm, kubernetes, etc). The Cluster object is given to the Client object when you create it.

Reply:
There is only one `Client` class in `dask.distributed`, which can be given the address of the scheduler it uses. If the address is not given, it'll create a local cluster based on the other keyword arguments. Looking at https://docs.dask.org/en/latest/setup/single-distributed.html the hand-created clusters can be given as the address kwarg to the client, and the `Client` docstring says:
```
address: string, or Cluster
    This can be the address of a ``Scheduler`` server like a string
    ``'127.0.0.1:8786'`` or a cluster object like ``LocalCluster()``
```

I guess if non-address non-local clusters are wanted, they could be defined via YAML magic in the `settings` dictionary.

Reply:
Yeah that might work. It might we hard to specify all of the `__init__` arguments for creating a cluster:

```
from dask_yarn import YarnCluster
from dask.distributed import Client

# Create a cluster where each worker has two cores and eight GiB of memory
cluster = YarnCluster(environment='environment.tar.gz',
                      worker_vcores=2,
                      worker_memory="8GiB")
# Scale out to ten such workers
cluster.scale(10)

# Connect to the cluster
client = Client(cluster)
```

But I see that some clusters can be configured by other YAML files (hadoop example shows that) so maybe that could be used too.

Anyway, I'm not sure there is a need for the class parameter here in this YAML then, right? At least not for the Client.